Physical_Engine/Query/BottomCentreline.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BH.Engine.Geometry;
+using BH.Engine.Reflection;
+using BH.Engine.Spatial;
+using BH.oM.Base;
+using BH.oM.Geometry;
+using BH.oM.Physical.FramingProperties;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Configuration;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Physical
+{
+    public static partial class Query
+    {
+
+        [Description("Returns the bottom centreline of an IFramingElement")]
+        [Input("BHoM.Physical.Elements.IFramingElement", "The IFramingEment to query the bottom centreline of")]
+        [Output("ICurve", "The bottom centreline of the IFramingElement")]
```

Comment:
Would change the name of the output to something closer to the name of a parameter (also needs a dot)

```suggestion
        [Output("curve", "The bottom centreline of the IFramingElement.")]
```